plugins/node/opentelemetry-instrumentation-mongodb/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "@opentelemetry/instrumentation-mongodb",
+  "version": "0.13.1",
+  "description": "OpenTelemetry mongodb automatic instrumentation package.",
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "repository": "open-telemetry/opentelemetry-js-contrib",
+  "scripts": {
+    "test": "nyc ts-mocha --parallel -p tsconfig.json 'test/**/*.test.ts'",
+    "codecov": "nyc report --reporter=json && codecov -f coverage/*.json -p ../../",
+    "tdd": "npm run test -- --watch-extensions ts --watch",
+    "clean": "rimraf build/*",
+    "lint": "eslint . --ext .ts",
+    "lint:fix": "eslint . --ext .ts --fix",
+    "precompile": "tsc --version",
+    "version:update": "node ../../../scripts/version-update.js",
+    "compile": "npm run version:update && tsc -p .",
+    "prepare": "npm run compile"
+  },
+  "keywords": [
+    "opentelemetry",
+    "mongodb",
+    "nodejs",
+    "tracing",
+    "profiling",
+    "plugin"
+  ],
+  "author": "OpenTelemetry Authors",
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=8.5.0"
+  },
+  "files": [
+    "build/src/**/*.js",
+    "build/src/**/*.d.ts",
+    "doc",
+    "LICENSE",
+    "README.md"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "devDependencies": {
+    "@opentelemetry/context-async-hooks": "^0.17.0",
+    "@opentelemetry/node": "0.17.0",
+    "@opentelemetry/tracing": "^0.17.0",
```

Comment:
```suggestion
    "@opentelemetry/tracing": "0.17.0",
```

Reply:
Same as express, not sure why you'd want to pin it ?